pinot-core/src/test/java/com/linkedin/pinot/query/aggregation/AggregationGroupByOperatorTest.java

```diff
@@ -79,21 +79,20 @@
 
 import junit.framework.Assert;
 
-
 public class AggregationGroupByOperatorTest {
   protected static Logger LOGGER = LoggerFactory.getLogger(AggregationGroupByOperatorTest.class);
   private final String AVRO_DATA = "data/test_sample_data.avro";
-  private static File INDEX_DIR = new File(FileUtils.getTempDirectory() + File.separator
-      + "TestAggregationGroupByOperator");
-  private static File INDEXES_DIR = new File(FileUtils.getTempDirectory() + File.separator
-      + "TestAggregationGroupByOperatorList");
+  private static File INDEX_DIR =
```

Comment:
Many of these seem like formatting changes? If so, could you please remove those changes?